static/js/biomedical/disease/drug_table.tsx

```diff
@@ -1,37 +1,59 @@
 import React from "react";
 
 import { GRAPH_BROWSER_REDIRECT } from "../bio_charts_utils";
-import { CompoundDiseaseTreatmentData } from "./chart";
-
-export interface DrugTableProps {
-  data: CompoundDiseaseTreatmentData[];
+export interface DrugTableColumnType {
```

Comment:
Let's keep everything consistent and also name this DrugTreatmentTableColumn

Reply:
Done!